webforces/server/tests.py

```diff
@@ -44,34 +43,219 @@ def test_cant_add_iden_users(self):
         self.assertEqual(self.core.db._getNextID("1u"), 1)
         self.assertEqual(self.core.db._getNextID("2u"), DBStatus.s_data_issue)
 
-    def test_can_get_user_by_correct_id(self):
+    def test_can_get_user_by_correct_user_id(self):
         user1 = User(0, "LOGIN_USER1", "FN_USER1", "SN_USER1", "MN_USER1", [])
-        status1, user1 = self.core.db.addUser(user1)
-        status2, user2 = self.core.db.getUserByID(user1.user_id)
+        user1 = (self.core.db.addUser(user1))[1]
```

Comment:
If you need to drop status it is more clear if you do the following thing:
```py
_, user1 = (self.core.db.addUser(user1))
```
Same for other cases.

Reply:
Let's put it off until refactoring the code.